CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+cmake_minimum_required(VERSION 3.1)
+
+project(kv_fdw)
+
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_C_STANDARD 99)
+
+#init pg enviroment
+execute_process(COMMAND pg_config --includedir-server OUTPUT_VARIABLE PG_INCLUDE OUTPUT_STRIP_TRAILING_WHITESPACE)
```

Comment:
Here, it makes an assumption that we have `pg_config` in the `$PATH`. For me, I need to update it manually to `/usr/pgsql-13/bin/pg_config` on centos 7.6.